modules/persistence/index.ts

```diff
@@ -0,0 +1,28 @@
+import AdmZip from 'adm-zip';
+import * as dotenv from 'dotenv';
+import minimist from 'minimist';
+import * as mongodb from 'mongodb';
+import { insertEntries } from './src/services/entries';
+import { insertMetadata } from './src/services/metadata';
+
+// Load command line args into a parameterized argv
+const argv = minimist(process.argv.slice(2));
+
+// Load .env, if present
+dotenv.config();
+
+const app = async (path) => {
```

Comment:
also are we not typing here? getting 'any' types pretty much in all persistence modules